app/lib/common/widgets/spaces/space_with_profile_card.dart

```diff
@@ -123,7 +129,12 @@ class SpaceWithAvatarInfoCard extends StatelessWidget {
         titleTextStyle: titleTextStyle,
         subtitleTextStyle: subtitleTextStyle,
         leadingAndTrailingTextStyle: leadingAndTrailingTextStyle,
-        title: Text(title, overflow: TextOverflow.ellipsis),
+        title: Row(
+          children: [
+            Expanded(child: Text(title, overflow: TextOverflow.ellipsis)),
+            if (showVisibilityMark) VisibilityChip(roomId: roomId),
+          ],
+        ),
         subtitle: buildSubtitle(context),
```

Comment:
I think we agreed to put the visibilty statement into the subtitle. the chip is just too much and too distracting there.

Reply:
Yes sorry, I forgot. let me do needful changes and share update.

Reply:
ehem... but that means for _suggested_ we now have _three_ rows? ... I think it would be more appropriate to put the visibility chip inside the `buildSubtitle(context)` widget .... just a `Suggested - Private` or something...